### ISSUE TITLE
Check logo

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/advocacy.mozilla.org#153]
It might be my eyes, but the word `Mozilla` seems too far above the word `Advocacy`. True or false?

<img width="463" alt="screenshot 2016-05-17 15 15 26" src="https://cloud.githubusercontent.com/assets/53411/15335989/61b38448-1c42-11e6-854f-b5f81044ba24.png">

**Comment by kristinashu** [comment]
Yeah we can tighten that up! We can do this update along with the upcoming OWF updates.

**Comment by kristinashu** [comment]
Move 'mozilla' and 'Advocacy' closer together by about 2 px. 

These are currently separate SVGs but let me know if it would be easier to have them as one?
